### DISCUSSION TITLE
Install specific version of boost (not working for dependencies)

### DISCUSSION THREAD

**Discussion by dsazonoff** [microsoft/vcpkg#24681] [Q&A] [answered]
Hello, I'm trying to install a specific version of some boost modules (ex 1.77). Problem that depended modules are installed with the latest version, what is not ok. Ofc, I can manually enumerate all possible dependencies but it's odd.
Here is my vcpkg.json:
```
{
  "name": "my-project",
  "version": "0.0.1",
  "homepage": "https://my-project",
  "dependencies": [
    {
      "name": "boost-beast",
      "version>=": "1.77.0"
    }
  ],
  "builtin-baseline": "b98afc9f1192becb2f447cee485ce36ba111f9f6",
  "overrides": [
    {
      "name": "boost-beast",
      "version": "1.77.0"
    }
  ]
}
```
It installs boost-beast v1.77, but also installs a lot of latest boost libraries with v1.79. Seems that "overrides" feature is not working for dependencies.

**Comment by autoantwort** [comment, marked as answer]
Maybe take a look at https://github.com/microsoft/vcpkg/blob/master/docs/examples/modify-baseline-to-pin-old-boost.md

**Reply by dsazonoff** [reply]
Yes, this is a possible workaround. But I'm wondering, is it an expected behavior?

**Reply by autoantwort** [reply]
I think so. The easiest thing is properly to use a baseline where boost is at version 1.77.
